virttest/utils_sriov.py

```diff
@@ -46,6 +47,40 @@ def find_pf(driver, session=None):
     return pf_tmp
 
 
+def get_pf_info(session=None):
+    """
+    Get PFs infomation
+
+    :param session: The session object to the host
+    :raise: exceptions.TestError when command fails
+    :return: dict, pfs' info
```

Comment:
It's better to have more detailed info about the return dict, like "pfs'info like {'3b:00.0': {'driver': 'ixgbe', 'pci_id': '0000:3b:00.0', 'iface': 'ens1f0', 'status': 'up'}, '3b:00.1': {'driver': 'ixgbe', 'pci_id': '0000:3b:00.1', 'iface': 'ens1f1', 'status': 'down'}}" just a small suggestion :-)

Reply:
Updated.